Frameworks/Foundation/NSURLSessionConfiguration.mm

```diff
@@ -136,4 +123,11 @@ + (NSURLSessionConfiguration*)backgroundSessionConfiguration:(NSString*)identifi
     return [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:identifier];
 }
 
+- (void)dealloc {
+    [_HTTPCookieStorage release];
+    [_URLCache release];
+    [_URLCredentialStorage release];
```

Comment:
I wouldn't use StrongId here. Since they're property getters and setters, you'd need to implement the backing ivar, any atomicity requirements, and the getter/setter yourself.